lib/checks/tables/th-has-data-cells.js

```diff
@@ -2,6 +2,11 @@ var tableUtils = axe.commons.table;
 var cells = tableUtils.getAllCells(node);
 var checkResult = this;
 
+var exemptRoles = ['presentation', 'none'];
+if (node.hasAttribute('role') && exemptRoles.includes(node.getAttribute('role'))) {
+	return true;
```

Comment:
These tables should be ignored, not passed. I suggest putting this in the selector or matcher instead.

Reply:
agree

Reply:
Ok, I've updated it. Thanks!